pages/renewal.tsx

```diff
@@ -2,19 +2,22 @@ import React from "react";
 import type { NextPage } from "next";
 import homeStyles from "../styles/Home.module.css";
 import styles from "../styles/search.module.css";
-import Renewal from "../components/domains/renewal";
+import RenewalV2 from "../components/domains/renewalV2";
+import { FormProvider } from "@/context/FormProvider";
 
 const RenewalPage: NextPage = () => {
   return (
-    <div className={homeStyles.screen}>
-      <div className={styles.container}>
-        <Renewal
-          groups={[
-            process.env.NEXT_PUBLIC_MAILING_LIST_GROUP_AUTO_RENEWAL ?? "",
-          ]}
-        />
+    <FormProvider>
```

Comment:
It's a detail but it makes more sense to not wrap the two divs screen and container and just take the register/renewal component.